### PR TITLE
log message if spawning bower fails

### PR DESCRIPTION
**Pull request by jasisk**
message

### CODE REVIEW

**Comment by jasisk** [comment]
Pending #190.

File: app/index.js [hunk 1]
```diff
@@ -164,7 +164,23 @@ module.exports = yeoman.generators.Base.extend({
...
             this.installDependencies({
-                skipMessage: true
+                skipMessage: true,
+                callback: function (err) {
+                    var errString;
+                    if (err) {
+                        if (err.code === 'ENOENT') {
+                            errString = err.code + ': ' + err.path;
+                            if (err.path === 'bower') {
+                                errString += ' - cannot locate ' + err.path + '.' +
```
Comment (grawk) [inline app/index.js]: So super minor but since we know `err.path === 'bower'` the errString can be defined more easily without the concatenation

Comment (jasisk) [inline app/index.js]: Magic number stuff. `¯\_(ツ)_/¯`

**Comment by grawk** [comment]
:+1:

**Pull request merged**